dojo/tools/api_msdefender/parser.py

```diff
@@ -0,0 +1,106 @@
+import json
+from dojo.models import Finding, Endpoint
+from .importer import MSDefenderApiImporter
+
+
+class ApiMSDefenderParser(object):
+    """
+    Import from MSDefender API /findings
+    """
+
+    def get_scan_types(self):
+        return ["MSDefender API"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "MSDefender API"
+
+    def get_description_for_scan_types(self, scan_type):
+        return ("MSDefender findings can be directly imported using the REST API")
+
+    def requires_file(self, scan_type):
+        return False
+
+    def requires_tool_type(self, scan_type):
+        return "MSDefender API"
+
+    def api_scan_configuration_hint(self):
+        return (
+            "Do not set anything here."
+        )
+
+    def get_findings(self, file, test):
+        if file:
+            data = []
+            vulnerabilityfile = json.load(file[0])
+            vulnerabilitydata = vulnerabilityfile['value']
+            data.append(vulnerabilitydata)
+            machinefile = json.load(file[1])
+            machinedata = machinefile['value']
+            data.append(machinedata)
```

Comment:
If it is "because of unit test", please drop this functionality and use `mock.patch` in unit tests
Example: https://github.com/DefectDojo/django-DefectDojo/blob/27e1ae609b90164a5d2a600d0a25919b9b60058a/unittests/tools/test_api_sonarqube_parser.py#L52-L56

Reply:
@kiblik could you resolve this as discussed in slack?

Reply:
Resolved as discussed in slack.